internal/acctest/localidentityidentityprofiles/local_identity_identity_profiles_test.go

```diff
@@ -0,0 +1,141 @@
+package acctest_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/acctest"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/provider"
+)
+
+const localIdentityIdentityProfilesId = "test"
+
+// Attributes to test with. Add optional properties to test here if desired.
+type localIdentityIdentityProfilesResourceModel struct {
+	id                  string
+	name                string
+	registrationEnabled bool
+	profileEnabled      bool
+}
+
+func TestAccLocalIdentityIdentityProfiles(t *testing.T) {
+	resourceName := "myLocalIdentityIdentityProfiles"
+	initialResourceModel := localIdentityIdentityProfilesResourceModel{
+		// Test is only run on attributes that do not require a PD dataStore.
+		id:                  localIdentityIdentityProfilesId,
+		name:                "example",
+		registrationEnabled: false,
+		profileEnabled:      false,
+	}
+	updatedResourceModel := localIdentityIdentityProfilesResourceModel{
+		id:                  localIdentityIdentityProfilesId,
+		name:                "example1",
+		registrationEnabled: false,
+		profileEnabled:      false,
```

Comment:
We still aren't covering any nested attributes here, which I think are important to test, like some of the config attributes. Also it's best if we change the values between the initialResourceModel and updatedResourceModel if possible.

Reply:
This resource would need a Directory dataStore to test the nested attributes.
Separate ticket opened to improve the test for nested attributes once we complete the DataStore resource.